### PR TITLE
Footer: added info to find the github repo for new contributors.

### PR DESCRIPTION
**Pull request by thekemkid**
This would resolve https://github.com/CoderDojo/community-platform/issues/753

Styling and placement of the message needs to be checked, I didn't have the dev environment on my machine to preview it.

### CODE REVIEW

File: web/locale/en_US/messages.po [hunk 1]
```diff
@@ -2624,3 +2624,6 @@ msgstr "Enabling this will require you or another admin with Ticketing Admin per
 
 msgid "Place name must be less than 40 characters."
 msgstr "Place name must be less than 40 characters."
+
+msgid "If you find any issues on this site, want to suggest features to the platform, or even better contribute, you can find all the relevant information over"
+msgstr If you find any issues on this site, want to suggest features to the platform, or even better contribute, you can find all the relevant information over"
```
Comment (tangentfairy) [inline web/locale/en_US/messages.po]: Missing a beginning quote here. Can we change the copy to "Zen is open source, you can contribute code and suggest features to help make it better" - where "contribute code" links to the community-platform repo and "suggest features" links to the issues section with the backlog label. :smiley_cat:

**Comment by thekemkid** [comment]
@tangentfairy Updated :)

File: web/public/css/footer.less [hunk 2]
```diff
@@ -29,7 +29,7 @@
   padding-top: 10px;
   float: right;
 }
-.cdf-info {
+.cdf-info .cp-info{
```
Comment (tangentfairy) [inline web/public/css/footer.less]: I think we're missing a comma after .cdf-info, otherwise only cp-info will be 15px and not the cdf-info class

**Comment by tangentfairy** [comment]
:+1: Looks good!

**Pull request merged**